### ISSUE TITLE
Implement accessibility features for menu-like elements

### ISSUE THREAD

**Issue by bugzilla-to-github** [mozilla/pontoon#2259]
*This issue was created automatically by a [script](https://github.com/mathjazz/bugzilla2github/).*
## [Bug 1690740](https://bugzilla.mozilla.org/show_bug.cgi?id=1690740)
Bug Reporter: @julen

Some of the popover menus currently in use in the Translate app (e.g. user profile or notifications menus) are missing basic accessibility features like keyboard navigation. This seems to be a regression because mainline Pontoon implements things like this already.
